README.md

```diff
@@ -1 +1,794 @@
-# Infrastructure
\ No newline at end of file
+# Infrastructure Repository
+
+This repository serves as an infrastructure example for using Docker Compose to deploy services such as Jitsi, WordPress, and Nextcloud. 
+It aims to provide a lightweight introduction to these tools and demonstrate the benefits of Infrastructure as Code.
+
+## Status
+⚠️ **Beta Status** ⚠️
+
+This repository is currently in beta status. Please note the following important considerations:
+- The nginx-example configuration requires special attention for SSL setup
+- SSL configuration depends on certificates that can only be created after enabling HTTP
+- Recommended approach: Create certificates → Enable SSL
+
+## Overview
+This repository provides a comprehensive infrastructure setup using Docker Compose to deploy various services including:
+- Jitsi (Video Conferencing)
+- WordPress (Content Management)
+- Nextcloud (File Sharing & Collaboration)
+- Monitoring Stack (Loki, Prometheus, Grafana)
+- Additional utilities (Watchtower, Jumphost)
+
+The goal is to demonstrate Infrastructure as Code practices while providing a lightweight, maintainable solution for self-hosted services.
+
+## Prerequisites
+
+### System Requirements
+- Linux server (tested with Ubuntu 24.04)
+- Internet domain with DNS control
+- Docker Compose v2.33.1 or later
+- Open firewall ports (see Firewall Configuration)
+
+### Required Docker Plugins
+- Loki logging plugin (required for container logging):
+```bash
+docker plugin install grafana/loki-docker-driver:latest --alias loki --grant-all-permissions
+```
+
+### Domain Configuration
+Ensure your domain's DNS settings are properly configured to point to your server's IP address.
```

Comment:
Should we add more information here? E.g.
This usually means to add a few entries to your DNS zone (with DOMAIN being your-new-domain.com and the SEVER_IP_ADDRESS you got from your server provider):
Name: DOMAIN, TTL: 86400, Type: A, Priority: 0, Data: SERVER_IP_ADDRESS
Name: www.DOMAIN, TTL: 86400, Type: A, Priority: 0, Data: SERVER_IP_ADDRESS
Name: *.DOMAIN, TTL: 86400, Type: A, Priority: 0, Data: SERVER_IP_ADDRESS